### ISSUE TITLE
How does this work?

### ISSUE THREAD

**Issue by bit-whisperer** [cellos51/InkAim#1]
I am trying to use the new cemu version but there is no documentation, so I am unsure how to get this working.

**Comment by cellos51** [comment]
If you're trying to compile from source you're out of luck (sorry). If you're just trying to use it normally, you just place the exe in your cemu folder, run it, and then when you're in game you can press f9 to lock the cursor (gyro aim has to be on) and left mouse is bound to shoot. If you need anything else don't be afraid to ask.

**Comment by bit-whisperer** [comment]
ah okay, I was just using the old inkaim (this repository) with the modded cemu version. The modded cemu version doesn't seem to do anything when I press F9 or F1.

**Comment by cellos51** [comment]
Yeah, I would really not recommend using this one because it is incredibly prone to drift. When I made these I didn't have any documentation because I planned this for friends to use only.

**Comment by bit-whisperer** [comment]
got it, you mean don't use the modded cemu version or don't use this old inkaim one?

**Comment by cellos51** [comment]
use cemu if you can. If you need anymore help let me know.

**Comment by bit-whisperer** [comment]
can I message you on discord or something?

**Comment by cellos51** [comment]
yeah. it's Elliptical#0513
